src/tasks/preset/client.tsx

```diff
@@ -126,7 +145,15 @@ const completeRender: TimedTask = timedTask({
 const group: TaskGroup = {
   groupId: 'Client',
   name: 'Client 👩‍💻',
-  tasks: [render, reRender, hydrate, domElementCount, domElementCountWithoutSvg, completeRender],
+  tasks: [
+    render,
+    reRender,
+    hydrate,
+    domElementCount,
+    domElementCountWithoutSvg,
+    reactFiberNodeCount,
```

Comment:
sadly because of the way import / export works right now, i think this task will need to be added last to the list (and also make it last defined)

We should fix that...

Reply:
Interesting! I'll make the change but I don't quite understand. Would be happy to look into fixing this ☺️